kythe/cxx/indexer/cxx/IndexerASTHooks.cc

```diff
@@ -603,11 +603,22 @@ const clang::Expr* IndexerASTVisitor::SkipTrivialAliasing(
       // star := *&var
       const auto* var = amp->getSubExpr()->IgnoreParens();
       if (const auto* dre = llvm::dyn_cast_or_null<clang::DeclRefExpr>(var)) {
-        return dre;
+        return std::make_pair(dre, std::nullopt);
```

Comment:
`make_pair` is almost always unnecessary and frequently results in an extra copy; just use {} on all return paths.